prototypes/hierarchy.py

```diff
@@ -157,9 +157,11 @@ def ensureOwnModuleRootNode(self):
 		:returns: The entity of the root-node.
 		:rtype: :class:`server.db.Entity`
 		"""
-		key = "rep_module_repo"
-		kindName = self.viewSkel().kindName + "_rootNode"
-		return db.GetOrInsert(key, kindName=kindName, creationdate=datetime.now(), rootNode=1)
+		return db.GetOrInsert(
+			db.Key(self.viewSkel().kindName + "_rootNode", "rep_module_repo"),
+			creationdate=datetime.now(),
+			rootNode=1
```

Comment:
Looks okay for me.
But I think would be more suitable 
```suggestion
			rootNode=True
```